app/config/custom-environment-variables.json

```diff
@@ -5,6 +5,10 @@
     "realm": "KC_REALM",
     "serverUrl": "KC_SERVERURL"
   },
+  "redis": {
```

Comment:
Nothing in readme to describe adding these to a config map?  When are we updating that?

Reply:
Readme has been updated